lua/damagelogs/server/discord.lua

```diff
@@ -11,6 +11,10 @@ local emitOnlyWhenAdminsOffline = Damagelog.DiscordWebhookMode == POST_MODES.WHE
 local limit = 5
 local reset = 0
 
+local use_chttp = pcall(require, "chttp")
+if use_chttp then
+    HTTP = CHTTP
+end
```

Comment:
This should fix #6 
I will need to update the wiki with install instructions.
I shall test this

Reply:
awesome